ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala

```diff
@@ -96,32 +74,38 @@ object Cli {
       .action(reportUsageOfDeprecatedOption("--target-port"))
       .hidden()
 
-    opt[String]("pem")
+    opt[File]("pem")
       .optional()
       .text("TLS: The pem file to be used as the private key. Applied to all endpoints.")
-      .action(pemConfig)
+      .action { (path: File, config: Config) =>
+        config.withTlsConfig(_.copy(keyFile = Some(path)))
+      }
 
-    opt[String]("crt")
+    opt[File]("crt")
       .optional()
       .text(
         """TLS: The crt file to be used as the cert chain.
-              |Required if any other TLS parameters are set. Applied to all endpoints.""".stripMargin
+          |Required if any other TLS parameters are set. Applied to all endpoints.""".stripMargin
       )
-      .action(crtConfig)
+      .action { (path: File, config: Config) =>
+        config.withTlsConfig(_.copy(keyCertChainFile = Some(path)))
+      }
 
-    opt[String]("cacrt")
+    opt[File]("cacrt")
       .optional()
       .text("TLS: The crt file to be used as the trusted root CA. Applied to all endpoints.")
-      .action(cacrtConfig)
+      .action { (path: File, config: Config) =>
+        config.withTlsConfig(_.copy(trustCertCollectionFile = Some(path)))
+      }
 
     opt[Double](name = "timeout-scale-factor")
       .optional()
       .action((v, c) => c.copy(timeoutScaleFactor = v))
       .text(
         """Scale factor for timeouts used in all test suites. Useful to tune timeouts
-              |depending on the environment and the Ledger implementation under test.
-              |Defaults to 1.0. Use numbers higher than 1.0 to make test timeouts more lax,
-              |use numbers lower than 1.0 to make test timeouts more strict.""".stripMargin
+          |depending on the environment and the Ledger implementation under test.
+          |Defaults to 1.0. Use numbers higher than 1.0 to make test timeouts more lax,
+          |use numbers lower than 1.0 to make test timeouts more strict.""".stripMargin
```

Comment:
Where are these indent changes come from? Do you use any special formatter?

Reply:
No, I'm using the defaults (afaict)

Reply:
@pbatko-da please check if you have "Scalafmt" under Preferences -> Editor -> Code style -> Scala -> Formatter.